pkg/network/http/ebpf_main.go

```diff
@@ -165,6 +166,11 @@ func (e *ebpfProgram) Init() error {
 	}
 	e.Manager.DumpHandler = dumpMapsHandler
 
+	numCPUs, err := utils.NumCPU()
```

Comment:
This is actually the wrong CPU count. We've raised this with CWS. You should use `cpupossible.Get()`

Reply:
TIL. Thanks for the heads-up